nixos/modules/services/misc/mbpfan.nix

```diff
@@ -82,8 +82,8 @@ in {
     environment = {
       etc."mbpfan.conf".text = ''
         [general]
-        min_fan_speed = ${toString cfg.minFanSpeed}
-        max_fan_speed = ${toString cfg.maxFanSpeed}
+        min_fan1_speed = ${toString cfg.minFanSpeed}
+        max_fan1_speed = ${toString cfg.maxFanSpeed}
```

Comment:
These shouldn't be added unless the options are set. Autodetection is probably better than risking specifying improper defaults.